src/otx/core/types/explain.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""OTX explain type definition."""
+
+from __future__ import annotations
+
+from enum import Enum
+
+
+class TargetExplainGroup(str, Enum):
+    """OTX target explain group definition.
+
+    Enum contains the following values:
+        IMAGE - This implies that single global saliency map will be generated for input image.
```

Comment:
Do I understand correctly that `Image` means activation map? But can we comment `Image` mentioning here since we don't have activation map hook for now?

Reply:
I will add in the very next PR. So let's keep it. To be honest, we do not have predictions as well yet